docker/api/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM meaningfy/rdf-validator-api:latest
+
+COPY resources/lam-skos-ap.ttl /usr/src/shacl-shapes/
```

Comment:
what I would do is:
Define an environment variable inside docker file for the rdf validator custom path and set `/usr/src/shacl-shapes/` as its value. 

Then copy the shacl shape file as you do. The difference is that the environment variable will be available by default in the Dockerfile.

Reply:
the same could be done for the port

Reply:
check how will that function in combination with the UI docker image